x-element.js

```diff
@@ -68,7 +87,9 @@ export default class XElement extends HTMLElement {
    * E.g., setup focus delegation or return host instead of host.shadowRoot.
    */
   static createRenderRoot(host) {
```

Comment:
With these changes — it’s far less likely that you will need to override the `createRenderRoot` method. IMO, that’s a good thing. As an author, I prefer to be able to declare things _in isolation_ and let the library cobble everything together.